### ISSUE TITLE
Item Embed widget doesn't display item

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vatican_exhibits#307]
Not sure if I'm missing something obvious but when I use the Item Embed widget on either our `vatican-dev` or BAV's `spotlight.bav` instance, the item itself is not displayed. I don't see an error in web inspector and the `<div />` for containing the item is there, but it is empty. This is not a problem in our normal Stanford exhibits.

Is this a known and expected limitation based on the way we're handling the Vatican manuscript objects? If so, we might want to investigate hiding the Item Embed widget from the widget selection panel, to prevent BAV curator confusion. If not, below are examples of the issue.

Here's what I get on `vatican-dev` (http://vatican-dev.sul.stanford.edu/palimpsests/about/pal-lat-24 red outline added to show where item should be):

<img width="898" alt="pal_lat_24___vatican_palimpsests_-_thematic_pathways_of_medieval_manuscripts" src="https://user-images.githubusercontent.com/101482/46827823-7f331f00-cd4e-11e8-8313-fddf33684959.png">

Here's the same text but using a Stanford exhibit (first widget in page at https://exhibits.stanford.edu/service-sandbox/feature/gary):

<img width="876" alt="gary___service_team_sandbox_-_spotlight_at_stanford" src="https://user-images.githubusercontent.com/101482/46827868-9a9e2a00-cd4e-11e8-86bc-cbaafebaf737.png">

**Comment by mejackreed** [comment]
We may have inadvertently done something to  override this behavior in vatican_exhibits. Do we need to reenable?

**Comment by ggeisler** [comment]
Not necessarily. They haven't mentioned wanting to use that widget, I don't think (it's been too long for me to remember what led me to try the widget on our server but I don't remember BAV saying they wanted to use it and it didn't work).

It might be helpful if we could just comment out the widget from the widget selection panel, if that is feasible, just so one of the BAV curators doesn't add it and not notice that it doesn't work, as in my top screenshot above.

I think commenting out the widget option from the panel might be the best approach, in case they later say they really really want it and we want to investigate getting it to work without having to add back removed code.
